sdk/schemaregistry/schema-registry-avro/package.json

```diff
@@ -3,44 +3,38 @@
   "version": "1.1.1",
   "description": "Schema Registry Avro Serializer Library with typescript type definitions for node.js and browser.",
   "sdk-type": "client",
-  "main": "dist/index.js",
-  "module": "dist-esm/src/index.js",
-  "browser": {
-    "./dist-esm/test/public/utils/env.js": "./dist-esm/test/public/utils/env.browser.js"
-  },
-  "types": "types/schema-registry-avro.shims.d.ts",
+  "main": "./dist/commonjs/index.js",
+  "module": "./dist/esm/index.js",
+  "browser": "./dist/browser/index.js",
+  "types": "./dist/commonjs/index.d.ts",
   "scripts": {
     "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
-    "build": "npm run clean && npm run bundle  && dev-tool run extract-api",
+    "build": "npm run clean && tshy && dev-tool run extract-api",
```

Comment:
could we use the new dev-tool command for building?

Reply:
Updated. I'll update the migration guide accordingly too

Reply:
uhhh I will owe you a coffee or something for updating the guide, I didn't even think of it 😄 

FWIW I did update the `migrate-package` command to do the right thing but have not merged the PR yet https://github.com/Azure/azure-sdk-for-js/pull/31269/files (and it wouldn't have mattered for you since you already started migrating)

Reply:
@maorleger I have been regularly doing my due diligence and I did put that command in the top of the page 🫡